default/pl/cat_printed.md

```diff
@@ -0,0 +1,151 @@
+
+Wyjątek stanowiłyby nowoczesne edycje włączone w proces katalogowania zbiorów o wartości historycznej, takie jak prywatna biblioteka osoby fizycznej. Tutaj RISM akceptowałoby rekordy muzyki współczesnej w celu udokumentowania całej kolekcji.
+
+RISM rozważy, oddzielnie rozpatrując każdy przypadek, czy dokumentować druki muzyczne wydane w XX wieku przez nieistniejących już wydawców, lub wydane przez mniejszych, regionalnych wydawców o znaczeniu historycznym.
+
+## **Kiedy należy wprowadzić nowy rekord (druki muzyczne)**
+
+RISM traktuje każdy rękopis jako pozycję unikalną i dlatego też każdy rękopis posiada swój własny rekord RISM. Natomiast materiały drukowane różnią się od rękopisów tym, że wiele egzemplarzy tego samego wydania można opisać za pomocą jednego rekordu (poprzez dodanie informacji o posiadanych zbiorach). Nowy rekord tworzy się dla wariantów istotnych.
+
+Poniżej zamieszczono wskazówki, które pomogą w ustaleniu, które warianty uzasadniają tworzenie nowego rekordu. (W tej części korzystano obszernie z dokumentu [Katalogowanie opisowe zbiorów specjalnych (zbiory muzyczne) [Descriptive Cataloging of Rare Materials (Music)]  ](http://http://rbms.info/dcrm/)).
+
+**Nowy rekord jest konieczny**, jeżeli opisywana pozycja wykazuje jedną lub więcej z następujących różnic w stosunku do rekordu  RISM:
+
+- **Inna treść**: Różnice w tytule lub autorach/współautorach. Strefa wydania, która wykazuje poprawki, zmiany, rozszerzenia, skróty, lub włączenie materiałów uzupełniających
+
+- **Inny wydawca.**
+-
+
+**Inny układ druku**: Zmiany w strefie objętości. Zmiana w formacie bibliograficznym lub muzycznym. Różnice w zakończeniach pięciolinii i w kustoszach (wykazane w wyniku porównania wielu egzemplarzy).
+
+-
+
+**Inny status wydania**: Usunięto lub wymieniono oryginalną stronę tytułową. Na oryginalnej strefie wydania, dystrybucji, produkcji itd. znajduje się nowa etykieta. Oryginalne wydanie nut wydano w okładce nowego wydawcy zawierającej więcej informacji niż podano na stronie tytułowej. Strona tytułowa serii jest nowa.
+
+-
+
+**Inna etykieta i/lub numery wydawcy.**
+
+**Nie należy tworzyć nowego rekordu,** jeżeli opisywana pozycja wykazuje jedną z poniższych różnic (jeżeli opisywana pozycja posiada _więcej niż jedną_ z poniższych różnic, należy zdecydować, czy nowy rekord jest konieczny.)
+
+-
```

Comment:
This list is also formatted incorrectly

Reply:
I just changed the file name here. I will edit in a separate phase.